File: Conversao Maiusculo e Minusculo/maiusminus.cs
using System;

class Program {
    static void Main(string[] args) {
        Console.Write("Digite um texto .....: ");
        string texto = Console.ReadLine();

        string textoMaiusculo = texto.ToUpper();
        string textoMinusculo = texto.ToLower();

        Console.Write("Texto em maiusculo ..: ");
        Console.WriteLine(textoMaiusculo);

        Console.Write("Texto em minusculo ..: ");
        Console.WriteLine(textoMinusculo);
    }
}

File: Conversao Maiusculo e Minusculo/maiusminus.pas
program TextCaseConverter;

var
  texto: string;
  textoMaiusculo, textoMinusculo: string;

begin
  Write('Digite um texto .....: ');
  ReadLn(texto);

  textoMaiusculo := UpCase(texto);
  textoMinusculo := LowerCase(texto);

  Write('Texto em maiusculo ..: ');
  WriteLn(textoMaiusculo);

  Write('Texto em minusculo ...: ');
  WriteLn(textoMinusculo);
end.
